src/components/Navigation/NavigationItem/_navigationItem.scss

```diff
@@ -77,18 +77,8 @@
         padding-bottom: 0.625rem;
       }
     }
-  }
-
-  &--end {
-    @include isHeaderMobile {
-      margin-top: $spacing-md;
-    }
-  }
-
-  &--start {
-    margin-right: 1rem;
 
-    & li:last-child {
+    & .view-all {
```

Comment:
hmm, this seems strange to have this as a special case and require a downstream consumer to pass the right class.  Maybe just remove the special styling and let the consumer override for the last link somehow?